tests/integration/test_query.py

```diff
@@ -173,10 +173,13 @@ def get_project_config():
     outcomes_consumer.assert_dropped_internal()
     assert request_count <= 30  # 30 secs to fetch, each request takes 1 second at least
 
-    for (_, error) in mini_sentry.test_failures:
+    for (_, error) in mini_sentry.test_failures[:-1]:
         assert isinstance(error, AssertionError)
         assert "error fetching project states" in str(error)
 
+    _, last_error = mini_sentry.test_failures[-1]
+    "failed to resolve project information" in str(last_error)
```

Comment:
This assertion was always broken. The last error generated by Relay has always been different, but we still saw the old messages in breadcrumbs. Now, we check for the correct terminal error.

Reply:
This expression is missing `assert ` in front.